src/components/EntityInfo/EntityMenu.vue

```diff
@@ -118,11 +151,15 @@ export default class EntityMenu extends Vue {
   @Getter(useConfigurationStore) getEditUrl!: string
   @Getter(useBusinessStore) getIdentifier!: string
   @Getter(useRootStore) getReasonText!: string
+  @Getter(useBusinessStore) isBenBcCccUlc!: boolean
+  @Getter(useBusinessStore) isCoop!: boolean
   @Getter(useBusinessStore) isFirm!: boolean
   @Getter(useBusinessStore) isGoodStanding!: boolean
   @Getter(useBusinessStore) isHistorical!: boolean
   @Getter(useRootStore) isPendingDissolution!: boolean
 
+  private expand = false
```

Comment:
I get a template warning when it uses a property or method that isn't public, so I've been removing the visibility word from those. Does this look OK for you in VS Code?

Reply:
hmm.. I do not get that warning. See below.  I can remove it though
![image](https://github.com/bcgov/business-filings-ui/assets/116035339/2762eef0-0b27-4bd1-ac2e-d3b2be08043d)

Reply:
I see the warning on the variable in the template.

Reply:
Oops...
![image](https://github.com/bcgov/business-filings-ui/assets/116035339/cf61557a-d06e-4c8b-be46-d768663f697e)

Reply:
Interesting. Do you have these extensions installed? Do you have others?

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/f111a2a2-197d-4da2-9a19-9ebf3c87fa28)

Reply:
Yes but It was not showing me any warning. Just to experiment, I reloaded and now it does show warning. I have always had this extension. 
I will remove 'private'

Reply:
OK.

(Sometimes the extension takes a few seconds (or longer) to parse the code and show warnings.)